pkg/crc/api/api_http_test.go

```diff
@@ -126,6 +126,21 @@ func TestHTTPApi(t *testing.T) {
 		},
 		configSetResult,
 	)
+	// Get the cpus config again after setting it to make sure it is set
+	// properly.
+	configGetAfterSetResult, err := client.GetConfig([]string{"cpus"})
```

Comment:
this is an API call that could make sense, but as mentioned in the PR description with a body message this makes not sense. This can only be coming as a querystring.